packages/webview_flutter/webview_flutter/test/webview_flutter_test.dart

```diff
@@ -126,25 +146,11 @@ void main() {
       'CACHE-CONTROL': 'ABC'
     };
     await controller!.loadUrl('https://flutter.io', headers: headers);
-    expect(await controller!.currentUrl(), equals('https://flutter.io'));
-  });
 
-  testWidgets("Can't go back before loading a page",
```

Comment:
This test was removed since it only tests the behavior of a fake Platform WebView and not the behavior of the WebView widget.